php-frontend/src/main/java/org/sonar/php/tree/visitors/PHPCheckContext.java

```diff
@@ -101,6 +101,11 @@ public FileIssue newFileIssue(PHPCheck check, String message) {
 
   @Override
   public File file() {
+    return file.file();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "file"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)